include/qdmi/common/enums.h

```diff
@@ -98,16 +98,19 @@ enum QDMI_DEVICE_STATUS_T {
 
 /// Enum of the site properties that can be queried.
 enum QDMI_SITE_PROPERTY_T {
-  QDMI_SITE_PROPERTY_TIME_T1 = 0, ///< `double` The T1 time of a site in µs.
-  QDMI_SITE_PROPERTY_TIME_T2 = 1, ///< `double` The T2 time of a site in µs.
+  /// `size_t` The id to identify the site in the circuit.
+  /// @details For example, `q[1]` in a QASM code identifies the site with id 1.
```

Comment:
I'd prefer to use the `/**` comment syntax for multi-line comments to stay consistent with the rest.

The details comment isn't quite accurate. If a quantum circuit defines a register `qreg p[1]` before the register declaration of `q`, all indices in q need to be shifted by 1.
I would rather like to draw the analogy with OpenQASM's physical qubits concept (https://openqasm.com/language/types.html#physical-qubits) or QIR's static addressing mode (casting ints to pointers).
An interesting bit in the OpenQASM spec is that IDs need not be consecutive.
I believe this might invalidate the test that you added here.

In my opinion, QDMI is not at all point where it could enforce QASM and QIR programs to be single-register. At least the current interface has no say in this.

Reply:
Good point, I extended the documentation here and mentioned hardware qubits both in QIR and QASM.